backend/src/bundles/users/user.service.ts

```diff
@@ -19,6 +20,30 @@ class UserService implements Service {
         return Promise.resolve(null);
     }
 
+    public async findByEmail(
+        email: UserEntity['email'],
+    ): Promise<UserGetOneItemResponseDto> {
+        const user = await this.userRepository.findByEmail(email);
```

Comment:
I guess this is similar to https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/88#discussion_r1305874025
```suggestion
        const user = await this.userRepository.find({email});
```